platform-sdk/platform-apps/demos/StatsDemo/src/main/java/com/swirlds/demo/stats/StatsDemoMain.java

```diff
@@ -139,7 +139,7 @@ public StatsDemoState newStateRoot() {
      */
     @Override
     public Function<VirtualMap, StatsDemoState> stateRootFromVirtualMap() {
-        throw new UnsupportedOperationException();
+        return (virtualMap) -> newStateRoot();
```

Comment:
This means, the virtual map is not used. Does it mean the app always starts from genesis?

What if there is an overloaded `newStateRoot(VirtualMap)` method, which calls `new StatsDemoState(virtualMap)`? Of course, `StatsDemoState` should be updated to extend from `VirtualMapState`, not `MerkleRootState`